ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2754,6 +2754,8 @@ tests:
   steps:
     cluster_profile: nutanix-qe-zone
     env:
+      COMPUTE_ZONE: failure-domain-1 failure-domain-2
+      CONTROL_PLANE_ZONE: failure-domain-2 failure-domain-3
```

Comment:
Currently we have two jobs are checking zone setting:
1. nutanix-ipi-zones-f14 (this one)
2. nutanix-ipi-zones-single-osimage-customized-f28 (checking single zone)

While per the code of `cucushift-installer-check-nutanix-zones`, actually there are 3 cases, one is default multi-zone, signle-zone and non-default multi-zone. Once we set `COMPUTE_ZONE` and `CONTROL_PLANE_ZONE` in this job, we would miss the coverage for `default multi-zone` case, right?

Reply:
Ah, or you meant we can use `nutanix-ipi-zones-f14-destructive` to cover the default multi-zone case.

Reply:
yes, I left default multi-zone to nutanix-ipi-zones-f14-destructive

Reply:
Also nutanix-ipi-zones-external-lb can cover default multi-zone case

Reply:
> Also nutanix-ipi-zones-external-lb can cover default multi-zone case

We did not inject cucushift-installer-check-nutanix-zones for the install workflow, right?

Reply:
Yes, what would you suggest? Let nutanix-ipi-zones-f14-destructive job to cover default multi-zone, or cover it in nutanix-ipi-zones-external-lb?

Reply:
I am okay with let nutanix-ipi-zones-f14-destructive job to cover default multi-zone. If you would like to use nutanix-ipi-zones-external-lb to cover it too, I am okay, too.